### PR TITLE
Improve BINDING.md: add LED video, add OLED option

### PR DESCRIPTION
**Pull request by olliw42**
Title says it

### CODE REVIEW

File: docs/BINDING.md [hunk 1]
```diff
@@ -6,11 +6,15 @@ For a Tx module and a receiver to be able to connect to each other, some paramet
 
 The binding procedure is as follows:
 
-1. On the Tx side you have three options: (i) press the bind button (for ca. 4 seconds) on the Tx module, (ii) initiate the binding via a CLI command, or (iii) initiate the binding via the mLRS configuration Lua script.
+1. On the Tx side you have three options: (i) press the bind button (for ca. 4 seconds) on the Tx module, (ii) initiate the binding via a CLI command, or (iii) initiate the binding via the mLRS configuration Lua script. If available, the binding can also be initiated via the OLED display.
```
Comment (jlpoltrack) [inline docs/BINDING.md:9]: Should the options be a nested list?

File: docs/BINDING.md [hunk 2]
```diff
@@ -6,11 +6,15 @@ For a Tx module and a receiver to be able to connect to each other, some paramet
...
 2. Press the bind button (for ca 4 seconds) on the receiver.
 
 The sequence doesn't matter, i.e., one also can first press the bind button on the receiver and then set the Tx module into binding mode.
 
 When in binding mode, the green and red LEDs will blink alternatively with ca 2.5 Hz.
 
-When a receiver is connected to a Tx module, the common parameters (as well as all other parameters) can be changed and be made active by issuing a Save either via the CLI or the mLRS configuration Lua script.
+When a receiver is connected to a Tx module, the common parameters (as well as all other parameters) can be changed and be made active by issuing a Save command either via the CLI or the mLRS configuration Lua script (or the OLED display if available).
```
Comment (jlpoltrack) [inline docs/BINDING.md:16]: Should mention the 'pstore' command for the CLI (instead of save).

Comment (olliw42) [reply docs/BINDING.md]: done, hopefully better now. More suggestiosn welcome :)

File: docs/BINDING.md (hunk 1 above)
Comment (olliw42) [reply docs/BINDING.md:9]: I'm ok with as it is. You sggest I should make it a nested list?

Comment (jlpoltrack) [reply docs/BINDING.md:9]: Okay, fine as is then.  :)

**Comment by jlpoltrack** [comment]
I think this is good to merge but do have one thought - should it be mentioned on this page that newly flashed hardware (or where a full chip erase was done) should bind automatically as they will have standard bind phrase?  I recall when Target had some issues and was trying to bind on newly flashed hardware when it shouldn't have been necessary...

**Comment by olliw42** [comment]
very good comment!
would you have a phrase in mind?

**Comment by jlpoltrack** [comment]
How about 'Note: when flashing mLRS to new hardware or after performing a full chip erase the Tx and Rx should bind automatically as they will both have the default bind phrase of 'mlrs.0'.'

**Comment by olliw42** [comment]
many THX.
The phrase couldn't be as clean. First off, it is actually not garanteed that a freshly flashed Tx and Rx will connect immediately, since e.g. defaults may be different and not only the bind phrase matters. Also, there is a mixup of language, the Tx and Rx do not bind, they need to have been bound in order to connect. :)

**Comment by olliw42** [comment]
note also that Brad suggested a way for a factory reset of the params. So while there is currently only the full chip erase option, there might be something coming in some not too distant feature (the current idea is to have some button sequence to do that, there might be also a command in the CLI)

**Comment by jlpoltrack** [comment]
Makes sense, will merge if you're all set.

**Pull request merged**